src/nvim/eval.c

```diff
@@ -12419,10 +12419,7 @@ static void f_jobstop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     (void)channel_close(data->id, kChannelPartRpc, &error);
   }
   process_stop((Process *)&data->stream.proc);
-  rettv->vval.v_number = 1;
-  if (error) {
-    EMSG(error);
-  }
+  rettv->vval.v_number = error == NULL ? 0 : 1;
```

Comment:
still needs to return the exit status? That would be in `data->stream.proc.status` or something like that.